dataeng/jobs/AnalyticsEmailOptinMaster.groovy

```diff
@@ -0,0 +1,105 @@
+        stringParam('CONFIG_FILENAME','default.yaml', 'Name of configuration file in analytics-secure/analytics-exporter.')
+        stringParam('OUTPUT_BUCKET','course-data/email-opt-in', 'Name of the bucket for the destination of the email opt-in data.')
+        stringParam('OUTPUT_PREFIX','email-opt-in-', 'Optional prefix to prepend to output filename.')
+        stringParam('NOTIFICATION_EMAILS','analytics@edx.opsgenie.net analytics-alerts@edx.org',
+                'Space separated list of emails to notify in case of failure.')
+        stringParam('DATE_MODIFIER','',
+                'Used to set the date of the CWSM dump.  Leave blank to use today\'s date.  Set to "-d 201x-0x-0x" if that is when the CWSM dump took place, typically the preceding Sunday.  (Leave off quotes.)')
+        stringParam('ORG_CONFIG','data-czar-keys/config.yaml', 'Path to the data-czar organization config file')
+        stringParam('DATA_CZAR_KEYS_BRANCH','master', 'Branch of the Data-czar-keys repository to use')
+    }
+
+    multiscm{
+        git {
+            remote {
+                url('git@github.com:edx/edx-platform.git')
+                branch('$PLATFORM_BRANCH')
+                credentials('1')
+            }
+            extensions {
+                pruneBranches()
+                relativeTargetDirectory('edx-platform')
+            }
+        }
+        git {
+            remote {
+                url('https://github.com/edx/edx-analytics-exporter.git')
+                branch('$EXPORTER_BRANCH')
+            }
+            extensions {
+                pruneBranches()
+                relativeTargetDirectory('analytics-exporter')
+            }
+        }
+        git {
+            remote {
+                url('git@github.com:edx-ops/analytics-secure.git')
+                branch('$SECURE_BRANCH')
+                credentials('1')
+            }
+            extensions {
+                pruneBranches()
+                relativeTargetDirectory('analytics-secure')
+            }
+        }
+        git {
+            remote {
+                url('git@github.com:edx/data-czar-keys.git')
+                branch('$DATA_CZAR_KEYS_BRANCH')
+                credentials('1')
+            }
+            extensions {
+                relativeTargetDirectory('data-czar-keys')
+            }
+        }
+    }
+
+    wrappers {
+        timestamps()
+    }
+
+    steps {
+        virtualenv {
+            nature("shell")
+            command(
+                readFileFromWorkspace("dataeng/resources/setup-platform-venv.sh")
+            )
+        }
+        virtualenv {
+            nature("shell")
+            command(
+                readFileFromWorkspace("dataeng/resources/setup-exporter-properties.sh")
+            )
+        }
+        downstreamParameterized {
+            trigger('analytics-email-optin-worker') {
```

Comment:
There are some updates to the xml job configs, most notably that the newer version of Jenkins requires that all parameters are declared on all child jobs (previous versions allowed you to pass in parameters and they were automatically included).  If you look at the current job configs on the new jenkins box you can see the changes I've made.